hypothesis-python/tests/nocover/test_from_type_recipe.py

```diff
@@ -0,0 +1,79 @@
+# coding=utf-8
+#
+# This file is part of Hypothesis, which may be found at
+# https://github.com/HypothesisWorks/hypothesis/
+#
+# Most of this work is copyright (C) 2013-2019 David R. MacIver
+# (david@drmaciver.com), but it contains contributions by others. See
+# CONTRIBUTING.rst for a full list of people who may hold copyright, and
+# consult the git log if you need to determine who owns an individual
+# contribution.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+#
+# END HEADER
+
+from __future__ import absolute_import, division, print_function
+
+import datetime
+import decimal
+import fractions
+import numbers
+import uuid
+
+import hypothesis.strategies as st
+from hypothesis import given
+from hypothesis.internal.compat import hbytes
+
+
+def everything_except(excluded_types):
+    """Recipe copied from the docstring of ``from_type``"""
+    return (
+        st.from_type(type)
+        .flatmap(lambda x: st.from_type(x))
+        .filter(lambda x: not isinstance(x, excluded_types))
+    )
+
+
+many_types = [
+    bool,
+    bytearray,
+    hbytes,
```

Comment:
This type is not registered under Python 2 - you *could* work around that by using `binary_type` here and `text_type` instead of `str`, but I'd just delete the `hbytes` line entirely.